data/generator.js

```diff
@@ -0,0 +1,116 @@
+    "state": "CA",
+    "zip_code": "94109",
+    "country": "US"
+  },
+"hours": [
+  {
+    "open": [
+      {
+        "end": faker.random.number({min:2000, max:2230}),
+        "day": 0,
+        "start": faker.random.number({min:630, max:1030})
+      },
+      {
+        "end": faker.random.number({min:2000, max:2230}),
+        "day": 1,
+        "start": faker.random.number({min:630, max:1030})
+      },
+      {
+        "end": faker.random.number({min:2000, max:2230}),
+        "day": 2,
+        "start": faker.random.number({min:630, max:1030})
+      },
+      {
+        "end": faker.random.number({min:2000, max:2230}),
+        "day": 3,
+        "start": faker.random.number({min:630, max:1030})
+      },
+      {
+        "end": faker.random.number({min:2000, max:2230}),
+        "day": 4,
+        "start": faker.random.number({min:630, max:1030})
+      },
+      {
+        "end": faker.random.number({min:2000, max:2230}),
+        "day": 5,
+        "start": faker.random.number({min:630, max:1030})
+      },
+      {
+        "end": faker.random.number({min:2000, max:2230}),
+        "day": 6,
+        "start": faker.random.number({min:630, max:1030})
+      }
+    ]
+  }
+],
+"categories": [
+  {
+    "type": faker.random.arrayElement(['American','Chinese','Mexican', 'Indonesian', 'Mongolian', 'Vietnamese']),
+  }
+],
+"From the Business": faker.lorem.paragraph(3),
+"more_info": {
+    "Has Soy-free Options": faker.random.boolean(),
+    "Liked by Vegetarians": faker.random.boolean(),
+    "Takes Reservations": faker.random.boolean(),
+    "Delivery": faker.random.boolean(),
+    "Take-out": faker.random.boolean(),
+    "Accepts Credit Cards": faker.random.boolean(),
+    "Accepts Apple Pay": faker.random.boolean(),
+    "Accepts Google Pay": faker.random.boolean(),
+    "Good For": faker.random.arrayElement(['breakfast', 'lunch', 'dinner']),
+    "Parking Street":faker.random.boolean(),
+    "Bike Parking":faker.random.boolean(),
+    "Wheelchair Accessible": faker.random.boolean(),
+    "Good for Kids": faker.random.boolean(),
+    "Good for Groups": faker.random.boolean(),
+    "Attire Casual": faker.random.arrayElement(['casual', 'sloppy', 'fancy']),
+    "Noise Level Average": faker.random.arrayElement(['quiet', 'average', 'loud']),
+    "Alcohol": faker.random.boolean(),
+    "Outdoor Seating":faker.random.boolean(),
+    "Wi-Fi": faker.random.boolean(),
+    "Has TV": faker.random.boolean(),
+    "Waiter Service": faker.random.boolean(),
+    "Caters": faker.random.boolean(),
+    "Gender Neutral Restroom": faker.random.boolean()
+}
+}
+    return restObj;
+}
+
+let dataGen = (int) => {
+    let dataArr = [];
+    let rest = null;
+    for (var i = 0; i < int; i++) {
+        rest = call(i);
+        dataArr.push(rest);
+    }
+    return dataArr;
+}
+
+let restaurantJSON = dataGen(100);
+restaurantJSON = JSON.stringify(restaurantJSON);
+
+let writeFileSync = promise.promisify(fs.writeFileSync);
```

Comment:
I think the standard for promised functions is to tag "async" on the end

Reply:
Took out the async call there, as it was unnecessary. I tagged another call in my database file